tensorflow_datasets/video/KTH_human_action_test.py

```diff
@@ -0,0 +1,24 @@
+import tensorflow as tf
+from tensorflow_datasets import my_dataset
+import tensorflow_datasets.testing as tfds_test
+
+
+class KTH_human_actionTest(tfds_test.DatasetBuilderTestCase):
+  DATASET_CLASS = my_dataset.KTH_human_action
+  SPLITS = {  # Expected number of examples on each split from fake example.
+      "train": 8,
+      "test": 8,
+      "validation":8
+  }
+  # If dataset `download_and_extract`s more than one resource:
+  DL_EXTRACT_RESULT = {
+      "walking": "/walking",  # Relative to fake_examples/my_dataset dir.
```

Comment:
you didn't include any files from "fake_examples" . did you forget to "git add" them ?